crates/chain/src/spk_txout_index.rs

```diff
@@ -52,6 +53,28 @@ impl<I> Default for SpkTxOutIndex<I> {
     }
 }
 
+impl<I: Clone + Ord + 'static> Indexer for SpkTxOutIndex<I> {
+    type Additions = ();
```

Comment:
Does `OwnedIndexer` also needs an implementation on `SpkTxOutIndex`? Seems like without that it can't be used in the `list_owned_*()` and `try_balance()` of `IndexedTxGraph`.